pkg/updatechecker/updatechecker.go

```diff
@@ -338,6 +340,49 @@ func ensureDesiredVersionIsDeployed(opts CheckForUpdatesOpts, clusterID string)
 	return nil
 }
 
+func getVersionToDeploy(opts CheckForUpdatesOpts, clusterID string, availableReleases []UpdateCheckRelease) UpdateCheckRelease {
```

Comment:
I wish we could just have the logic for deployLatestVersion etc handle this
but that all runs after this returns